server/libs/infra/src/repositories/media.repository.ts

```diff
@@ -22,11 +28,31 @@ export class MediaRepository implements IMediaRepository {
         return;
 
       case 'jpeg':
-        await sharp(input, { failOnError: false })
-          .resize(options.size, options.size, { fit: 'outside', withoutEnlargement: true })
-          .jpeg()
-          .rotate()
-          .toFile(output);
+        if (!options.raw) {
```

Comment:
It might be nice to put these into their own functions. Should make for easier unit testing and may be easier to read.

```ts
class {
  resize()
  resizeRaw()
}
```

Reply:
Yes, agree. I was trying to see if the approach was even worthwhile before going through the extra effort of splitting it out, mocking it, etc., but I think it's going to work so that's a good suggestion.